sdk/analytics/src/test/java/com/klaviyo/analytics/KlaviyoTest.kt

```diff
@@ -61,34 +61,35 @@ internal class KlaviyoTest : BaseTest() {
     }
 
     private val capturedProfile = slot<Profile>()
-    private val apiClientMock: ApiClient = mockk<ApiClient>().apply {
+    private val mockApiClient: ApiClient = mockk<ApiClient>().apply {
```

Comment:
Naming convention fix, sometimes using mock as a prefix, sometimes as a suffix. Suggesting we consolidate to prefix.